suites/upgrade/hammer/2-workload/rbd.yaml

```diff
@@ -0,0 +1,14 @@
+workload:
+   sequential:
+   - workunit:
+       clients:
+         client.0:
+           - rbd/import_export.sh
+       env:
+         RBD_CREATE_ARGS: --new-format
+   - print: "**** done rbd/import_export.sh 2-workload"
+   - workunit:
```

Comment:
can run these in parallel. test_cls_rbd won't work if it's upgraded before it starts, since hammer has new class methods it'll try to test